### PR TITLE
Output parser for our linter

### LINKED ISSUE #2: Add tldr-lint output parser
It should just produce simple array of objects with error descriptions, error codes and line numbers.

### PR DESCRIPTION
**Pull request by EmilyGraceSeville7cf**
closes #2

Please review this PR carefully as I am not JS developer.

### CODE REVIEW

**Comment by EmilyGraceSeville7cf** [comment]
Interesting... 🤔 

image

File: extension.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// Your extension is activated the very first time the command is executed
+
+/**
+ * @param {vscode.ExtensionContext} context
+ */
+function activate(context) {
+
+	// Use the console to output diagnostic information (console.log) and errors (console.error)
+	// This line of code will only be executed once when your extension is activated
+	console.log('Congratulations, your extension "tldr-ide" is now active!');
```
Comment (navarroaxel) [inline extension.js:15]: I think 2 spaces indentation is better

File: extension.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	// The command has been defined in the package.json file
+	// Now provide the implementation of the command with  registerCommand
+	// The commandId parameter must match the command field in package.json
+	let disposable = vscode.commands.registerCommand('tldr-ide.helloWorld', function () {
```
Comment (navarroaxel) [inline extension.js]:
```suggestion
	let disposable = vscode.commands.registerCommand('tldr-ide.helloWorld', () => {
```

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=733558
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"dbaeumer.vscode-eslint"
+	]
+}
```
Comment (navarroaxel) [inline .vscode/extensions.json]:
```suggestion
}

```

**Comment by EmilyGraceSeville7cf** [comment]
Closed, no more work in progress.

**Pull request closed without merging**